services/compute/virtualmachine/virtualmachine.go

```diff
@@ -564,8 +588,23 @@ func (c *client) getVirtualMachineStorageProfileOsDisk(d *wssdcloudcompute.Disk)
 	if d == nil {
 		return &compute.OSDisk{}
 	}
+	var managedDisk *compute.VirtualMachineManagedDiskParameters
+	if d.ManagedDisk != nil {
+		managedDisk = &compute.VirtualMachineManagedDiskParameters{}
+		if d.ManagedDisk.SecurityProfile != nil {
+			var securityEncryptionType compute.SecurityEncryptionTypes
```

Comment:
Should we set to type `None` by default?

Reply:
sure, I'll set it to none by default for consistency, although the type already defaults to none